packages/suite/src/support/messages.ts

```diff
@@ -275,46 +275,126 @@ export default defineMessages({
         id: 'TR_EXCHANGE_OFFERS_REFRESH',
     },
     TR_EXCHANGE_FOR_YOUR_SAFETY: {
-        defaultMessage: 'Keep your security in mind',
+        defaultMessage: 'Exchange {fromCrypto} to {toCrypto} with {provider}',
```

Comment:
Changes for already existing keys has to be applied directly in Crowdin otherwise they are overriden with next Crowdin sync.

Reply:
All translations keys were changed.